_layouts/page.html

```diff
@@ -35,9 +35,15 @@ <h4 class="usa-accordion__heading desktop:display-none">
       id="crt-page--content"
       class="mobile-lg:grid-col {% if page.sidenav %}tablet:grid-col-12 desktop:grid-col-8{% endif %}"
     >
-      {% if page.disclaimer %} {% include alert.html title="This is a test site. Do not rely on the
-      information provided." text="This webpage is a prototype meant for user research. It has not
-      undergone final review for legal accuracy and is not intended to provide legal guidance." %}
+    {% if page.title =='Page not found' %}
+      <div id="redirect-pending">
+        {% include alert.html type="info" text="We're checking the archive.ADA.gov site for you, please wait." %}
+      </div>
+    {% endif %}
```

Comment:
hmm, why move this from 404.html?

Could you instead make this a block and extend it in 404.md, to ensure we don't try to redirect on other pages?

Reply:
I have it here because it needs to appear above the page title and this was the only way I found to do that. Is there a different way I'm missing?

Reply:
The equivalent in django would be sections/blocks, where you can say:

_layouts/page.html: "this is the alert block, it is empty"
404.html: "here's what should go in the alert block"

Reply:
interesting - that looks like the liquid way to do it, yeah. Thanks for that!